update-controlled-shapes.py

```diff
@@ -0,0 +1,84 @@
+import logging
+import typer
+from rdflib.namespace import RDF, RDFS, VOID, XSD, SOSA, DCTERMS, PROV, TIME, SH
+from rdflib import Graph, Namespace, URIRef, Literal, BNode
+
+app = typer.Typer()
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+@app.command()
+def fetch(filename: str):
+
+    g = Graph()
+
+    g.parse(filename)
+
+    q_delete = """
+        PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+        PREFIX sh: <http://www.w3.org/ns/shacl#>
+        delete {
+            ?uri sh:property ?property .
+            ?property sh:in ?node .
+            ?node rdf:rest ?item .
+            ?item rdf:first ?head ;
+                rdf:rest ?tail .
+            ?property sh:path ?path .
+        }
+        where {
+            ?uri sh:property ?property .
+            ?property sh:in ?node .
+            ?node rdf:rest* ?item .
+            ?item rdf:first ?head ;
+                rdf:rest ?tail .
+            ?property sh:path ?path .
+        }
+    """
+
+    logger.info(f"Delete sh:in in file '{filename}'")
+    g.update(q_delete)
```

Comment:
Use rdflib.collection.Collection to delete the list of items.

See https://rdflib.readthedocs.io/en/stable/apidocs/rdflib.html#rdflib.collection.Collection